### PR TITLE
Add mobile apps section

### PR DESCRIPTION
**Pull request by philipecosia**

### CODE REVIEW

File: en.md [hunk 1]
```diff
@@ -56,7 +56,52 @@ We set one or several persistent cookies in order to save the user’s settings
...
+
+Additionally to the data described under 7.g. we collect the following data when you use our app:
+
+ - Usage statistics of the app, e.g. which interface elements are interacted with
+ - Data relating to your app or your end device, e.g. the name of the device, its manufacturer, model, operating system number, app number or SDK version and the device number (IMEI)
+
+The Ecosia app may use your locational data in anonymized, respectively pseudonymized, form if you have allowed access to these data on your device. We use the locational data in order to give you a better Ecosia experience or to show more relevant search results or advertising to you. Personal data such as names, addresses or dates of birth will not be collected, stored or transmitted.
+
+These data are anonymised after a short space of time, generally 7 days. The anonymised data are saved for as long as is necessary for the evaluation. The data are not transferred to any third party. We will not access your data located outside of the app, such as your calendar, photos, messages or such like unless you have permitted us to do so.
```
Comment (jasongwartz) [inline en.md]: The word "data" is only singular in this context, so this should read "This data is anonymised".

Comment (jasongwartz) [reply en.md]: Also, in the second sentence: "The anonymised data is saved".
Third sentence: "The data is not transferred"

Comment (philipecosia) [reply en.md]: @jasongwartz Argh thats what it read first and then Trudie changed it into plural when proofreading it. Well at least its just a grammar change no content change that needs to be reflected in the translations.

Comment (jasongwartz) [reply en.md]: Ok. I can talk to Trudie about it, but I am 10000% sure.

Comment (philipecosia) [reply en.md]: @jasongwartz No need it also sounds better to me. Changed it.

**Review by jasongwartz — APPROVED** [review]

**Pull request merged**